Remotion/Web/Development.WebTesting/Accessibility/AccessibilityResultViolationsExtensions.cs

```diff
@@ -126,6 +126,21 @@ public static class AccessibilityResultViolationsExtensions
       return violations.Where(v => v.Rule.ID != ruleID || !ArrayEquals(v.TargetPath.Select(p => p.XPath).ToArray(), xPath)).ToArray();
     }
 
+    /// <summary>
+    /// Removes elements where known workarounds cause violations.
+    /// </summary>
+    public static IEnumerable<AccessibilityRuleResult> IgnoreKnownIssues (
+        [NotNull] this IEnumerable<AccessibilityRuleResult> violations)
+    {
+      //RM-8984 determine how this should be handled
```

Comment:
Question: since we now know how this is handled, do you want to update the TODO comments with the offical ticket and a note that the KnownIssues code is removed again with the resolution of the new jira ticket for ListMenu?